server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyServerRequest.scala

```diff
@@ -24,6 +24,7 @@ case class NettyServerRequest(req: HttpRequest, attributes: AttributeMap = Attri
     QueryParams.fromMultiMap(multiMap)
   }
   override lazy val method: Method = Method.unsafeApply(req.method().name())
+  override lazy val showShort: String = s"$method ${req.uri()}"
```

Comment:
we don't have to strip anything, or will this include the scheme  & fragment here?

also, maybe worth adding some docs to showShort or ServerRequest that impls should override for performance?

Reply:
I ran some tests on our Netty servers (and Vert.X, which is based on Netty), and `req.uri` returned only path + query params, without fragment, user info, etc.
Good idea to add a comment about overriding showShort, I'll do it.